### PR TITLE
Made changes to Documentation based on latest master.

### PR DESCRIPTION
**Pull request by NasserFlexCompute**

### CODE REVIEW

File: docs/source/solverConfiguration/solverConfiguration.rst [hunk 1]
```diff
@@ -268,7 +268,8 @@ turbulenceModelSolver
    +--------------------------------+-----------------+-------------------------------------------------------------------------------------------------------------------------------------------+
    | equationEvalFrequency          |        4        | Frequency at which to evaluate the turbulence equation in loosely-coupled simulations                                                     |
    +--------------------------------+-----------------+-------------------------------------------------------------------------------------------------------------------------------------------+
-   | kappaMUSCL                     |        -1       | Kappa for the muscle scheme, range from [-1, 1] with 1 being unstable.                                                                    |
+   | gradientReconstructionLimiter  |        1.0      | The strength of gradient limiter used in reconstruction of solution variables at the faces (specified in the range [0.0, 2.0]).           |
+   |                                |                 | 0.0 and 2.0 correspond to the highest (lowest) and lowest (highest) levels of limiting (accuracy), respectively.                          |
```
Comment (chaitan3) [inline docs/source/solverConfiguration/solverConfiguration.rst:272]: Maybe we should specify that 2 results in second order reconstruction and 0 is first order reconstruction

File: docs/source/capabilities/userDefinedDynamics.rst [hunk 2]
```diff
@@ -16,7 +16,7 @@ The complete case configuration file can be downloaded here: `Flow360.json <http
 
 Some of the parameters are dicussed in more detail here:
 
-1. **dynamicsName**: This is an arbitrary name selected by the user for the dynamics. The results of each user-defined dynamics are saved to *dynamicsName_v2.csv*.
+1. **dynamicsName**: This is an arbitrary name selected by the user for the dynamics. The results of each user-defined dynamics are saved to *udd_dynamicsName_v2.csv*.
```
Comment (chaitan3) [inline docs/source/capabilities/userDefinedDynamics.rst:19]: remove arbitrary from this sentence

**Review by chaitan3 — DISMISSED** [review]

**Review by will-flex — APPROVED** [review]

File: docs/source/capabilities/userDefinedDynamics.rst [hunk 3]
```diff
@@ -16,7 +16,7 @@ The complete case configuration file can be downloaded here: `Flow360.json <http
 
 Some of the parameters are dicussed in more detail here:
 
-1. **dynamicsName**: This is an arbitrary name selected by the user for the dynamics. The results of each user-defined dynamics are saved to *dynamicsName_v2.csv*.
+1. **dynamicsName**: This is a name selected by the user for the dynamics. The results of each user-defined dynamics are saved to *udd_dynamicsName_v2.csv*.
```
Comment (will-flex) [inline docs/source/capabilities/userDefinedDynamics.rst:19]: The results of each user-defined dynamics are saved to *udd_dynamicsName_v2.csv* ->
The results of each user-defined dynamic is saved to *udd_dynamicsName_v2.csv* -> OR
The results of user-defined dynamics are saved to *udd_dynamicsName_v2.csv*

**Review by chaitan3 — DISMISSED** [review]

File: docs/source/solverConfiguration/solverConfiguration.rst [hunk 4]
```diff
@@ -1,464 +1,467 @@
...
-========================
-
-.. _wall_vel:
-
-1. a NoSlipWall boundary with a prescribed velocity
-
-.. literalinclude:: sampleJsons/noSlipWall_velocity.json
-   :linenos:
-
+Solver Configuration
```
Comment (chaitan3) [inline docs/source/solverConfiguration/solverConfiguration.rst:1]: Why is this diff so large?

**Review by chaitan3 — APPROVED** [review]

**Pull request merged**